### ISSUE TITLE
Support for CentOS targets

### ISSUE THREAD

**Issue by phahulin** [poanetwork/deployment-playbooks#90]
Deployment playbooks support one base OS image at this time - Ubuntu 16.04. This issue requests support for CentOS 7.X as base image for POA nodes.

To add CentOS 7.x support:

- [x] Add CentOS base image to the local development environment
- [X] Add CentOS base image to the Terraform scripts
- [x] Change packages deployment method from `apt` to `yum` based on `ansible_os_family` variable
- [x] Fix issues with different package names on CentOS and Ubuntu
- [x] Fix issues with different configuration for the services on CentOS and Ubuntu
- [x] Verify changes by deploying nodes to CentOS and Ubuntu vms

Example of maintaining different set of tasks for different platforms:

```
# file: tasks.yml

- import_tasks: tasks_ubuntu.yml
  when: ansible_os_family == "Debian"

- import_tasks: tasks_centos.yml
  when: ansible_os_family == "RedHat"
```

**Comment by phahulin** [comment]
@Ykisialiou yes, it's possible. In general, our intent is to make network as diversified as possible in terms of operating systems, hosting providers, etc. Besides these parameters, also
`PARITY_BIN_LOC`, `PARITY_BIN_SHA256`, `ORCHESTRATOR_BIN_LOC`, `ORCHESTRATOR_BIN_SHA256` will be different.
Alexey initially proposed [in this PR](https://github.com/poanetwork/deployment-playbooks/pull/91/files#diff-f5053687c683779ecc5daafbb4ebd341) putting host in two groups simultaneously, one for network role and one for os-role, like so 
```
[bootnode]
192.0.2.1

[centos]
192.0.2.1
```

**Comment by Ykisialiou** [comment]
Looks like can be closed, #144 fixed this
